### DISCUSSION TITLE
make:controller --requests doesn't work

### DISCUSSION THREAD

**Discussion by TheArKaID** [laravel/framework#39917] [Q&A] [answered]
I use Laravel Framework 8.74.0, and try to create a Controller with Resource, Requests, API and Test options.
As a result, Resource, API, and Test options implemented successfully, but I just got `Controller created successfully.` and `Test created successfully.` responses, there's no something like `Request created successfully.` (just like when we create Request with `php artisan make:request`. Here's what I tried.

![image](https://user-images.githubusercontent.com/35050722/144967273-302cd0fb-ca21-4ed3-a92d-bd219fcf22e8.png)

*I delete the Controller and Test before running another make command

I could create Requests with `php artisan make:request`, just trying to use this `--requests` option.

Anyone could tell me what's wrong ?

**Comment by ikudosi** [comment]
Are you ensuring "--requests" is a valid option on that Laravel version? Please do "php artisan make:controller --help" to see all valid options.

**Reply by TheArKaID** [reply]
I'll check it later. But, as [mentioned here](https://laravel.com/docs/8.x/controllers#generating-form-requests) , it's `--requests`

**Comment by TheArKaID** [comment, marked as answer]
Hmm...okay, so, the [docs](https://laravel.com/docs/8.x/controllers#generating-form-requests) didn't mentioned it clearly. If we want to use `--requests` or `-R` in the `make:controller` command, we need to use the `--model` option. It's mentioned [here](https://laravel.com/docs/8.x/controllers#specifying-the-resource-model) but specifically for route model binding.
That's all.
